typescript/infra/scripts/verify.ts

```diff
@@ -32,7 +32,7 @@ async function main() {
   // from solidity/core/hardhat.config.ts
   const compilerOptions: CompilerOptions = {
     codeformat: 'solidity-single-file',
-    compilerversion: 'v0.8.13+commit.07a7930e',
+    compilerversion: 'v0.8.16+commit.07a7930e',
```

Comment:
we should just add an override flag for this too

Reply:
yeah probably, but leave that for a future PR